src/entries/contentScript/views/CheckSafety.svelte

```diff
@@ -0,0 +1,64 @@
+<svelte:options tag="check-safety"/>
+
+<script lang="ts">
+    import {onMount} from "svelte";
+    import {goplus} from "~/entries/contentScript/views/network";
+
+    let currentUrl = window.location.href;
+
+    let isLoading = true;
+    let isAudited = null;
+    const ERROR_CODE = 2026;
+    const SUCCESS_CODE = 1;
+
+    const checkSafeteCurrentUrl = async () => {
+        isLoading = true;
+
+        goplus
```

Comment:
we should move this code into background script to make sure it can request on any website